addon/webextension/content-script.js

```diff
@@ -18,6 +22,8 @@ myPort.onMessage.addListener(function(m) {
   }
 });
 
+const SUPPORTURL = `https://support.mozilla.org/en-US/lookingglass`
```

Comment:
This currently 404s, but I trust will be live in time since I see how it's used below.
(nit: Also lacks a semi-colon and can probably use double quotes instead of template backticks)

Reply:
Good point! Fixing.